packages/swirl-components/src/components/swirl-table-row-group/swirl-table-row-group.tsx

```diff
@@ -43,7 +70,9 @@ export class SwirlTableRowGroup {
             )}
           </span>
         </div>
-        <slot></slot>
+        <div class={{ "rows-container": true, "is-hidden": !this.isVisible }}>
```

Comment:
Issue: For consistency BEM-style class names would be nice. So:
`rows-container` -> `table-row-group__rows-container` 
and
`is-hidden` -> `table-row-group__rows-container--is-hidden`

Reply:
Good point. Your review is encouraging me to develop more in swirl. I am learning a lot.